i18n/en.json

```diff
@@ -5404,7 +5404,7 @@
   },
   {
     "id": "model.user.is_valid.position.app_error",
-    "translation": "Invalid position: must not be longer than 35 characters."
+    "translation": "Invalid position: must not be longer than 128 characters."
```

Comment:
@jasonblais Currently in production the validation message returns "Invalid position: must not be longer than 35 characters." However, it's actually allowing 64 characters (if you, say, make a request with curl). Do we want to keep the validation message consistent with the UI (soft) validation (ie 35) or change it to be consistent with the DB limit (ie 128)? I have no preference I just want to make sure there's awareness of this difference.

Reply:
@mkraft Is this same error message used in the UI?

Reply:
@jasonblais I'm not sure if the UI shows the message. You would have to change the markup to allow the input to be able to see it in the UI.

Reply:
@mkraft Oh, right. Because we automatically prevent users from typing more than 35 characters.

Should be fine then.